addons/sale/models/sale.py

```diff
@@ -1134,10 +1134,7 @@ def product_id_change(self):
                 self.product_id = False
                 return result
 
```

Comment:
We call a method on sale order line instead of directly calling get_product_myultine so that is the description can depends on other field of the sale order line, a freature used by event tickets